### PR TITLE
Create form.txt

### PR DESCRIPTION
**Pull request by NguyenHieu71**

### CODE REVIEW

File: form.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+username
```
Comment (NguyenTrongCuong) [inline form.txt:1]: bad naming

**Review by NguyenTrongCuong — CHANGES_REQUESTED** [review]
fixing needed

**Comment by NguyenHieu71** [comment]
i have fixed it, go check it bro

**Review by NguyenTrongCuong — APPROVED** [review]
good

**Pull request merged**